docs/reference/configuration/data_types.rst

```diff
@@ -569,51 +569,9 @@ group.
 General data: GEN_DATA
 ~~~~~~~~~~~~~~~~~~~~~~
 
-The ``GEN_DATA`` keyword is used to load *arbitrary* which has been generated
-by the forward model. ERT does not have any awareness of the type of data
-encoded in a ``GEN_DATA`` keyword; it could be the result of gravimetric
-calculation or the pressure difference across a barrier in the reservoir. This
-means that the ``GEN_DATA`` keyword is extremely flexible, but also slightly
-complicated to configure. Assume a ``GEN_DATA`` keyword is used to represent the
-result of an estimated position of the oil water contact which should be
-compared with a oil water contact from 4D seismic; this could be achieved with
-the configuration:
-
-::
-
-	GEN_DATA 4DWOC  RESULT_FILE:SimulatedWOC_%d.txt  INPUT_FORMAT:ASCII   REPORT_STEPS:0
-
-The ``4DWOC`` is an arbitrary unique key, ``RESULT_FILE:SimulatedWOC%d.txt``
-means that ERT will look for results in the file ``SimulatedWOC_0.txt``. The
-``INPUT_FORMAT:ASCII`` means that ERT will expect the result file to be
-formatted as an ASCII file.
-
-The ``REPORT_STEPS:0`` is tightly bound to the ``%d`` integer format specifier
-in the result file - at load time the ``%d`` is replaced with the integer values
-given in the ``REPORT_STEPS:`` option, for the example given above that means
-that ``%d`` will be replaced with ``0`` and ERT will look for the file
-``SimulatedWOC_0.txt``. In principle it is possible to configure several report
-steps like: ``REPORT_STEPS:0,10,20`` - then ERT will look for all three files
-``SimulatedWOC_0.txt, SimultedWOC_10.txt`` and ``SimulatedWOC_20.txt``. It is
-quite challenging to get this right, and the recommendation is to just stick
-with *one* result file at report step 0 [#]_, in the future the possibility to
-load one keyword ``GEN_DATA`` for multiple report steps will probably be
-removed, but for now the ``GEN_DATA`` configuration is *quite strict* - it will
-fail if the ``RESULT_FILE`` attribute does not contain a ``%d``.
-
-.. [#] The option is called *report step* - but the time aspect is not really
-       important. You could just as well see it as an arbitrary label, the only
-       important thing is that *if* you have a corresponding ``GEN_OBS``
-       observation of this ``GEN_DATA`` vector you must match the report step
-       used when configuring the ``GEN_DATA`` and the ``GEN_OBS``.
-
-Observe that since the actual result file should be generated by the forward
-model, it is not possible for ERT to fully validate the ``GEN_DATA`` keyword
-at configure time. If for instance your forward model generates a file
-``SimulatedWOC_0`` (without the ``.txt`` extension you have configured), the
-configuration problem will not be detected before ERT eventuallly fails to load
-the file ``SimulatedWOC_0.txt``.
-
+The ``GEN_DATA`` keyword is used to load *arbitrary* files which have been generated
```

Comment:
Remove arbitrary and be more specific? Must be text files.

Reply:
Sure 👍